spring/src/main/java/com/vitech/archtests/spring/rules/TransactionalRules.java

```diff
@@ -43,4 +45,36 @@
       .should()
       .bePublic()
       .as("@Transactional methods should be public because no proxy will be used then.");
+
+  /**
+   * Transactional should be used in Spring beans.
+   * Transactional management - https://docs.spring.io/spring-framework/docs/4.2.x/spring-framework-reference/html/transaction.html
+   */
+  @ArchTest
+  public static final ArchRule METHODS_MARKED_TRANSACTIONAL_MUST_BE_LOCATED_IN_SPRING_BEAN = methods()
```

Comment:
Shouldn't we also implement `CLASSES_MARKED_TRANSACTIONAL_MUST_BE_LOCATED_IN_SPRING_BEAN` test?

Reply:
Done